tools/scoring/tests/parse_config_test.py

```diff
@@ -57,6 +60,28 @@ def testAppendTypes(self, mock_print):
     ]
     mock_print.assert_has_calls(calls)
 
+  def testMatchReportingEntities(self):
+    proposed = validator.Deserialize(
+        ['tests/fixtures/files/proposed/match_reporting_entities.yaml'])[0]
```

Comment:
https://github.com/google/digitalbuildings/blob/master/tools/scoring/tests/samples/proposed/building_config_example.yaml

Reply:
As above; I think it's logically easier to map what is being tested to the more concise file.